collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,291 @@
+
+Rendering contributors have been relentless in identifying and implementing performance enhancements this release cycle. Many areas of the renders, both 2D and 3D have been optimized and we expect that many users will notice an improvement to performance this release cycle. Especially for 2D lighting, rendering on mobile devices, and general 2D rendering. 
+
+### XR
+Thanks to Godot’s unique feature of it's editor being made with the engine itself, we’ve been able to bring the Godot editor to unconventional places, such as the web and Android devices. Building upon the latter, Fredia Huya-Kouadio completed the proof of concept started by Bastiaan Olij years ago, to add support for using the Android editor on XR devices using OpenXR ([GH-96624](https://github.com/godotengine/godot/pull/96624))! You can test the current version by sideloading the APK, currently supported on Meta Quest 3 or Quest Pro.
+
+
+## Changelog
+
+**TODO contributors** submitted **TODO improvements** for this release. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#4.4-beta1) for the complete list of changes since the 4.4-dev7 snapshot. You can also review [all changes included in 4.4](https://godotengine.github.io/godot-interactive-changelog/#4.4) compared to the previous 4.3 feature release.
+
+This release is built from commit [`TODO`](https://github.com/godotengine/godot/commit/TODO).
+
+## Downloads
+
+{% include articles/download_card.html version="4.4" release="beta1" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- .NET 8.0 or newer is required for this build, changing the minimal supported version from .NET 6 to 8.
+
+{% include articles/prerelease_notice.html %}
+
+## Known issues
+
+During the beta stage, we focus on solving both regressions (i.e. something that worked in a previous release is now broken) and significant new bugs introduced by new features. You can have a look at our current [list of regressions and significant issues](https://github.com/orgs/godotengine/projects/61) which we aim to address before releasing 4.4. This list is dynamic and will be updated if we discover new showstopping issues after more users start testing the beta snapshots.
+
+With every release, we accept that there are going to be various issues which have already been reported but haven't been fixed yet. See the GitHub issue tracker for a complete list of [known bugs](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+label%3Abug+).
+
+- Baking a Lightmap3D is more prone to crash https://github.com/godotengine/godot/issues/101391
```

Comment:
```suggestion
- Baking a Lightmap3D is more prone to crashing https://github.com/godotengine/godot/issues/101391
```

Reply:
Whoops, I meant to come back and finish that sentence